build-logic/jvm/src/main/kotlin/gradlebuild/jvm/argumentproviders/CiEnvironmentProvider.kt

```diff
@@ -49,7 +49,7 @@ class CiEnvironmentProvider(private val test: Test) : CommandLineArgumentProvide
     private
     fun collectMirrorUrls(): Map<String, String> =
         // expected env var format: repo1_id:repo1_url,repo2_id:repo2_url,...
-        System.getenv("REPO_MIRROR_GRDEV_URLS")?.ifBlank { null }?.split(',')?.associate { nameToUrl ->
+        System.getenv("REPO_MIRROR_URLS")?.ifBlank { null }?.split(',')?.associate { nameToUrl ->
```

Comment:
Do we need to do this change on `master`, `release` and `release6x` again? It is also unclear to me if that will break older versions on CI again.

Reply:
I suggest doing it on release, master and release6x, so it's consistent. `REPO_MIRROR_GRDEV_URLS` was just an intermediate variable as far as I understand.
The Gradle build seems to be configured in a why that it won't break even if the variable is abscent, so I don't think we will break anything.